source/_posts/2021-04-09-android-april-2021-release.markdown

```diff
@@ -0,0 +1,71 @@
+
+As of this release the app now has a total of 71 sensors, given your device supports them all your number may be less. That is a lot of sensors and we don't expect a lot of users to actually use them all. We have added 2 new features to make this page easier to navigate. You can now filter by showing only the enabled sensors to get rid of the sensors you don't want to use, if you have all sensors enabled don't expect to see this filter. You can also perform a search against the list of sensors to find one quickly and manage it.
+
+Notification History now lets you search by the `message` that was sent to the device. Searching will bypass the filter options we have, that limit the view to the last 100 notifications. Filtering and delete options have been moved to the top right hand corner to make better use of the space.
+
+<p class='img'>
+<img src='/images/blog/2021-04-09-android-april-2021/action_bar.png' alt='Screenshot of Sensor Search and Filter'></a>
+Screenshot of Sensor Search & Filter.
+</p>
+
+## Shortcuts
+
+Sometimes when you open the app you may already know exactly where you want to go. You may find yourself on the same view often to see your cameras or give the nursery a quick look to make sure things are ok. You may even want to quicky see the history of an entity like when the door was last opened. In this release we have introduced [Android Shortcuts](https://developer.android.com/guide/topics/ui/shortcuts) which will let you quickly get to anywhere in the Home Assistant frontend directly from your home screen. Shortcuts will look like a separate app on the home screen allowing you to make them easily accessible and even place them into a folder for better organization. You can navigate to any Lovelace [view or dashboard](https://www.home-assistant.io/lovelace/dashboards-and-views/) including other pages like the [Shopping List](https://www.home-assistant.io/integrations/shopping_list/). You can also navigate to any entity directly to get more information like the history or see the graph.
+
+There are a few different shortcut types and in the nature of Home Assistant we have opted not to add static shortcuts because they are static! We support dynamic shortcuts which will show up under the app long press menu, once you create a shortcut you will be able to drag it on to your home screen. There is also support for pinned shortuts that can be added automatically to your home screen without needing to drag the icon, given your device and launcher support pinned shortcuts. Checkout the [documentation](https://companion.home-assistant.io/docs/integrations/android-shortcuts) for more details including known limitations.
```

Comment:
```suggestion
There are a few different shortcut types and in the nature of Home Assistant, we have opted not to add static shortcuts because they are static! We support dynamic shortcuts which will show up under the app long-press menu. once you create a shortcut, you will be able to drag it onto your home screen. There is also support for pinned shortcuts that can be added automatically to your home screen without needing to drag the icon, given your device and launcher support pinned shortcuts. Check out the [documentation](https://companion.home-assistant.io/docs/integrations/android-shortcuts) for more details including known limitations.
```